src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -65,125 +65,6 @@ public override string ToString()
     }
         ";
 
-        private static readonly string trivalRemainingTuples = @"
```

Comment:
Testing, we should still support and test against definitions in source as well as definitions in metadata.